packages/apollo-server-express/src/index.ts

```diff
@@ -22,4 +22,4 @@ export {
 export { graphqlConnect, graphiqlConnect } from './connectApollo';
```

Comment:
Doesn't a-s-e need to export gql now too?

Reply:
It does now https://github.com/apollographql/apollo-server/pull/1161/files#diff-cbf9fd9271eaacde30d8349212808beeR2 🎉